FishjamClient/src/main/java/org/membraneframework/rtc/InternalMembraneRTC.kt

```
@@ -210,6 +212,7 @@ internal class InternalMembraneRTC(
     }
 
     fun updateEndpointMetadata(endpointMetadata: Metadata) {
+        if (!canUpdateMetadata)return
```

also when we cannot update metadata we just fail silently?